test/uk/gov/ons/sbr/models/legalUnit/LegalUnitSpec.scala

```diff
@@ -0,0 +1,54 @@
+package uk.gov.ons.sbr.models.legalUnit
+
+import org.scalatest.{ FreeSpec, Matchers }
+import play.api.libs.json.Json
+import support.JsonString.{ int, optionalString, string, withValues }
+import support.sample.SampleLegalUnit
+import uk.gov.ons.sbr.models.legalunit.LegalUnit
+
+class LegalUnitSpec extends FreeSpec with Matchers {
+
+  private trait Fixture extends SampleLegalUnit {
+    def expectedJsonStrOf(legalUnit: LegalUnit): String =
+      s"""
+         |{${
+        withValues(
+          string("uBRN", legalUnit.uBRN.value),
```

Comment:
In the Json representations, I believe our convention is lowercase - ubrn / ubrnref seem preferable.